pageserver/src/http/routes.rs

```diff
@@ -84,11 +63,11 @@ fn get_config(request: &Request<Body>) -> &'static PageServerConf {
 
 // Helper functions to construct a LocalTimelineInfo struct for a timeline
 
-fn local_timeline_info_from_loaded_timeline(
-    timeline: &LayeredTimeline,
+fn build_timeline_info(
+    timeline: &Arc<LayeredTimeline>,
```

Comment:
Should work without Arc. Since Arc<T> [implements](https://doc.rust-lang.org/std/sync/struct.Arc.html#impl-Deref) Deref::Target = T passing &timeline at call site should do the trick